public/v4/apps/yourls.yml

```diff
@@ -0,0 +1,82 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: mysql:$$cap_database_version
+        volumes:
+            - $$cap_appname-db-data:/var/lib/mysql
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD: $$cap_db_pass
+            MYSQL_DATABASE: $$cap_YOURLS_DB_NAME
+            MYSQL_USER: $$cap_db_user
+            MYSQL_PASSWORD: $$cap_db_pass
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-db
+        image: yourls:$$cap_yourls_version
+        volumes:
+            - $$cap_appname-logs:/usr/src/yourls/logs
```

Comment:
Not familiar with yourls - but is mapping the logs as a persistent volume really necessary?

Reply:
You are right, not necessary!